<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleTableItem</class>
 <widget class="QWidget" name="RoleTableItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>66</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>66</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>66</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="roleTableItemBg">
     <property name="styleSheet">
      <string notr="true">#roleTableItemBg{
    background-color : #FFFFFF;
    border : 1px solid #EBEDF0;
}

*{
     font-family : 微软雅黑;
     font-size : 14px;
     line-height : 19px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="idLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>174</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="roleLabel">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>0</y>
        <width>174</width>
        <height>66</height>
       </rect>
      </property>
      <property name="text">
       <string>超级管理员</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailLabel">
      <property name="geometry">
       <rect>
        <x>348</x>
        <y>1</y>
        <width>220</width>
        <height>64</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#emailLabel{
    border : none;
}</string>
      </property>
      <property name="text">
       <string>569334855@qq.com</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>568</x>
        <y>0</y>
        <width>150</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>管理员</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="statusButton">
      <property name="geometry">
       <rect>
        <x>777</x>
        <y>18</y>
        <width>72</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#statusButton{
     color : #3686FF;
     background-color : #EBF3FF;
     border : none;
     border-radius : 10px;
     font-size : 12px;
     line-height : 16px;
}</string>
      </property>
      <property name="text">
       <string>启用</string>
      </property>
     </widget>
     <widget class="QLabel" name="commentLabel">
      <property name="geometry">
       <rect>
        <x>906</x>
        <y>0</y>
        <width>188</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>超管账号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="editBtn">
      <property name="geometry">
       <rect>
        <x>1115</x>
        <y>25</y>
        <width>30</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#editBtn{
    background-color : transparent;
    border : none;
    color : #3686FF;
}</string>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delBtn">
      <property name="geometry">
       <rect>
        <x>1175</x>
        <y>25</y>
        <width>30</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#delBtn{
    background-color : transparent;
    border : none;
    color : #FD6A6A;
}</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
